monkey/common/cloud/aws_service.py

```diff
@@ -1,8 +1,26 @@
 import boto3
+import botocore
```

Comment:
Added this to requirements?

Reply:
Now I did.
To avoid this in future, I also created #294